openapi.yaml

```diff
@@ -4075,14 +4085,16 @@ components:
         The billing plan to process and charge the job or service with.
 
         Billing plans MUST be accepted in a *case insensitive* manner.
-        If `null` is specified by the client explicitly,
-        the server MUST store the default plan and persist it regardless of any
-        changes to the default billing plan in the future. If given, the default
-        plan specific to the user has higher priority than the default plan of
-        the back-end.
-
-        The list of plans and the default plan for the back-end can be retrieved
-        from `GET /`. The default plan for the user can be retrieved from `GET /me`.
+        Back-ends MUST resolve the billing plan in the following order:
+        1. Use the `plan` that has been provided in the request, if a non-null value is present.
+
+        If `null` has been provided for the `plan`:
+        2. Use the `default_plan` exposed through `GET /me`, if available.
+        3. Otherwise, use the `default_plan` exposed through `GET /`.
```

Comment:
What if it's still unresolved after `3`?

Reply:
If billing is supported, an error should be thrown. Added the BillingPlanMissing error.